Specs/Data/Cesium3DTiles/Metadata/ImplicitHeightSemantics/makeMetadata.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
```

Comment:
I moved these scripts to a branch of the `samples-generator`: https://github.com/CesiumGS/3d-tiles-validator/blob/bounding-volume-semantics/samples-generator/lib/, I'll remove them from this PR.